### PR TITLE
add some sample launch files

### PR DESCRIPTION
**Pull request by devoncarew**
@sethladd - here are some possible launch files. They'll all need `path`s, so we know what files they apply to. And the launch `type` will tell us which launch handler to use.

The other info in there is for the use of the launch types, and will vary between types. The `pub` / `test` one is for a launch type we don't currently support - it's just in there to play with the format.

### CODE REVIEW

File: .atom/launches/utils_test.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Pub launch configuration for test/utils_test.dart.
+type: pub
```
Comment (sethladd) [inline .atom/launches/utils_test.yaml]: does this mean `pub run` ? Maybe we want to specify that clearly.

File: .atom/launches/tooltest.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Command-line launch configuration for tool/test.dart.
+type: cli
```
Comment (sethladd) [inline .atom/launches/tooltest.yaml]: Do we want to specify which versions of these types are compatible?

**Comment by sethladd** [comment]
Very cool, thanks for sharing.

File: .atom/launches/utils_test.yaml (hunk 1 above)
Comment (devoncarew) [inline .atom/launches/utils_test.yaml]: `pub_run`? makes sense to me

File: .atom/launches/tooltest.yaml (hunk 2 above)
Comment (devoncarew) [inline .atom/launches/tooltest.yaml]: Do you mean versioning the file?

Comment (sethladd) [inline .atom/launches/tooltest.yaml]: Sorry, I meant version of the cli or flutter.

File: .atom/launches/utils_test.yaml (hunk 1 above)
Comment (sethladd) [inline .atom/launches/utils_test.yaml]: sgtm

File: .atom/launches/tooltest.yaml (hunk 2 above)
Comment (devoncarew) [inline .atom/launches/tooltest.yaml]: My sense is that that is orthogonal to this file.
